submissions/franchukv/js-oop-frogger/app.js

```diff
@@ -0,0 +1,136 @@
+const CANVAS_HEIGHT = 606;
+const CANVAS_WIDTH = 505;
+const TILE_HEIGHT = 83;
+const TILE_WIDTH = 101;
+
+const PLAYER_CONF = {
+    size: {
+        height: 40,
+        width: 40
+    },
+    position: {
+        x: 202,
```

Comment:
You could calculate this from TILE_WIDTH programmatically. So whenever tile width changes, this param changes accordingly and automagically.